### DISCUSSION TITLE
Is it possible to have media and hardware stats both work in the chat?

### DISCUSSION THREAD

**Discussion by ProjecttFrostt** [VolcanicArts/VRCOSC#50] [Q/A] [answered]

**Comment by VolcanicArts** [comment, marked as answer]
Yes.

If you turn off `Continuous Show` in the Media module's settings, Media will override the ChatBox for however long `Display Period` is set whenever a new song plays and then return control back to Hardware Stats.

If you decide you want to show the current time and duration of the song requiring you to keep `Continuous Show` on, then pausing the song will cause Hardware Stats to once again have control over the ChatBox.
